src/views/Teams/EditTeam.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useState } from 'react'
+import { useHistory, useParams } from 'react-router-dom/cjs/react-router-dom.min'
+import { getTeamById, updateTeamById } from '../../services/teams'
+
+export default function EditTeam() {
+
+    const [name, setName] = useState('')
+    const [city, setCity] = useState('')
+    const [state, setState] = useState('')
+    const [loading, setLoading] = useState(true)
+    const {id} = useParams()
+    const history = useHistory()
+
+    const handleEdit = async (e) => {
+        e.preventDefault()
+        const response = await updateTeamById(id, { name, city, state })
+        history.push(`/team/${response[0].id}`)
+    }
+
+    useEffect(() => {
+        async function getTeamDeets() {
+            const response = await getTeamById(id)
+            setName(response.name)
+            setCity(response.city)
+            setState(response.state)
+            setLoading(false)
+        }
+        getTeamDeets()
+    }, [id]
+    )
+
+    return (
+        <div>
+            {loading ? <h1>'loading'</h1>
+                    : <>
+                <fieldset>
+                    <legend>Edit Team</legend>
+                    <form onSubmit={handleEdit}>
+
+                        <label htmlFor='teamName'>Team Name</label>
+                        <input 
+                        aria-label='team name'
+                        type='text' 
+                        id='teamName' 
+                        name='name'
+                        value={name}
+                        onChange={(e) => setName(e.target.value)}
+                        />
+
+                        <label htmlFor='city'>City</label>
```

Comment:
fyi, when your label is adjacent to an input you want to associate it with, you can just wrap the input in the label tag so you don't need to mess with the input name matching the htmlFor